tutorials/katacoda/getting-started/pipeline.md

```diff
@@ -0,0 +1,58 @@
+You can now build the two tasks into a Tekton pipeline. Open and edit
+`tutorials/katacoda/getting-started/src/tekton-katacoda/pipelines/pipelineTemplate.yaml`.
```

Comment:
should be `website/tutorials/...` based on where it was cloned in the setup step.

same in a few other places